x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -47,6 +47,11 @@ export const LATEST_FINDINGS_INDEX_DEFAULT_NS =
   'logs-cloud_security_posture.findings_latest-default';
 export const LATEST_FINDINGS_RETENTION_POLICY = '26h';
 
+export const KSPM_RETENTION_POLICY = '5h';
+export const CSPM_RETENTION_POLICY = '24h';
+export const VULN_MGMT_RETENTION_POLICY = '96h';
+export const ALL_RETENTION_POLICY = '72h';
```

Comment:
I realized that I don't really understand what 'all' means, even after looking in the code. I don't think it's related to the PR though, more of a general observation. But maybe you'll have some ideas how to document in a way that it explains that, for example near the `POSTURE_TYPE_ALL` definition

Reply:
all just means  all (any of kspm/cspm/vuln_mgmt)

Reply:
> all just means all (any of kspm/cspm/vuln_mgmt)

`any` would probably be a better wording than `all`

Reply:
> all just means all (any of kspm/cspm/vuln_mgmt)
that I understand, what I don't understand the logic behind it. Why the retention for all or any is different from each one of the posture types, what's the logic behind it?